common/lib/xmodule/xmodule/js/src/video/08_video_speed_control.js

```diff
@@ -30,12 +30,21 @@ function (Iterator) {
     SpeedControl.prototype = {
         template: [
             '<div class="speeds menu-container">',
-                '<a class="speed-button" href="#" title="',
-                    gettext('Speeds'), '" role="button" aria-disabled="false">',
-                    '<span class="label">', gettext('Speed'), '</span>',
+                '<button class="control speed-button" aria-label="',
+                    gettext('Speed: Press UP to enter the speed menu then use the UP and DOWN arrow keys to navigate the different speeds. Press ENTER to change to the selected speed.'),
+                    '" aria-disabled="false" aria-expanded="false" role="application">',
```

Comment:
Since we use the arrow keys to navigate the menus, we need to tell the screen readers to let keypresses not be stopped by the screenreader and go through to the application (or control). This allows the arrow keys to navigate the menu, rather than navigate the site, as is the case with JAWS.